text/0026-activation-distribution.md

```diff
@@ -0,0 +1,116 @@
+# Project title
+- Start Date: 2020/04/27
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/26
+
+## Decision, including impact on distributions
+OpenMRS Microfrontends will have their own default activator functions.
+Microfrontends will be identified as such by their names being suffixed with `-app`.
+Microfrontends' will provide activator and lifecycle functions in their exports according
+to a specific microfrontend interface, defined in  this RFC. The interface bundle will be
+dynamically split from the main application bundle via a dynamic import in the lifecycle function.
+The app shell will load the app shell config if available. The app shell will register all
+microfrontends in the import map with Single SPA, and start the application.
+
+The existing concept of a "root config" will be superseded by the app shell
+and an app shell config.
```

Comment:
```suggestion
The existing concept of a "root config" will be superseded by the initial script,
which is optionally configurable by a new kind of "root config."
```

Reply:
@FlorianRappl Shoot, just this one more, and then that's it!

Reply:
Awesome thanks for the enhancement @brandones ! :rocket: